app.js

```diff
@@ -106,22 +120,46 @@ function displayFriends() {
         friendEl.addEventListener('click', () => {
             // > handle the three possible outcomes:
             // 1. No mushrooms, set a message to go hunt for more
-            // 2. Friend is already fully satisfied (3), set a message to pick another friend
-            // 3. Feed friend mushroom:
-            // a. "pop" a mushroom off the mushrooms array
-            // b. increase friend.satisfied by 1
-            // c. set a message that the friend enjoyed the mushroom,
-            //    include the friend name and mushroom type in the message
-
+            if (mushrooms.length === 0) {
+                message = `You need more mushrooms to feed your hungry friends!`;
+                
+
+            } else if (friend.satisfied === 3) {
+                message = `${friend.name} is full and happy, send them on their way.`;
```

Comment:
using the state of an object to post a message then using that object to populate the text